README.md

```diff
@@ -27,20 +27,20 @@ Each job can be configured with one Gerrit server.
 
 
 # Environments
-* `linux`
-    * `java-1.6`
-        * `maven-3.0.4`
-* Java 1.7: Not officially supported, but if you prefer you can use it
-  for development. Patches to fix problems with Java 1.7 are welcome.
+* Supported deployment runtime: minimum Java 1.6
```

Comment:
I'm not sure this follows the [literate spec](https://github.com/jenkinsci/literate-plugin) It might work. Have you tested it?